### ISSUE TITLE
Add support for more compressing filetypes

### ISSUE THREAD

**Issue by gabrielsimoes** [SimpleMobileTools/Simple-File-Manager#93]
I think it would be really nice to support .rar, .7z, .bzip,... filetypes in addition to .zip files (already supported)

**Comment by Phoenix1747** [comment]
Yes! Especially RAR support would be awesome, encountered this problem today, had to transfer the files to my PC :/

**Comment by elyorch76** [comment]
Yesss please, very interested in this

**Comment by illeatmyhat** [comment]
couldn't decompress a RAR today
